docs/user-guide/job/expansions/README.md

```diff
@@ -0,0 +1,194 @@
+---
+---
+
+* TOC
+{:toc}
+
+# Example: Multiple Job Objects from Template Expansion
+
+In this example, we will run multiple Kubernetes Jobs created from
+a common template.  You may want to be familiar with the basic,
+non-parallel, use of [Job](../jobs.md) first.
+
+## Basic Template Expansion
+
+First, download the following template of a job to a file called `job.yaml.txt`
```

Comment:
I named it this because it is not quite a yaml file, it is a file that is intended to be preprocessed before it becomes valid yaml.